packages/docs/community/release-process/blockchain-client.md

```diff
@@ -0,0 +1,179 @@
+# Blockchain Client Release Process
+
+{% hint style="warning" %}
+This release process is a work in progress. Many infrastructure components required to execute it are not in place, and the process itself is subject to change.
+{% endhint %}
+
+## Versioning
+
+Releases of celo-blockchain are numbered according to semantic versioning, as described at [semver.org](https://semver.org).
+
+  * New releases of celo-blockchain can be expected as follows:
+  * Major releases: approximately yearly
+  * Minor releases: approximately 4 times a year
+  * Patch releases: as needed
+
+All builds are identified as `unstable` (a development build) or `stable` (a commit released as a particular version number). There should only ever exist one commit with a version `x.y.z-stable` for any `(x, y, z)`.
+
+### Signatures
+
+Artifacts produced by this build process (e.g. tags, binaries, Docker images) will be signed. Signatures are produced using any one of the core developer keys listed below.
+
+{% hint style="warning" %}
+Work in progress: Core developer keys are not yet hosted.
+{% endhint %}
+
+Public keys for these developers are hosted on celo.org and can be imported to `gpg` with the following command:
+
+```bash
+gpg --auto-key-locate wkd --locate-keys $EMAIL
```

Comment:
What do you think about using the same signature scheme on the EVM vs. gpg? I know gpg of course is the more standardized toolchain for virtually all other software, but there is something nice about having one consistent scheme in the Celo ecosystem?

Reply:
A) `gpg` supports ECC signatures, although I am not sure if it supports `secp256k1`.
B) Unless we want to built a lot of tooling ourselves, it's really best to use something standardized, and nothing else is as standardized as `gpg`. Even if we did end up building our own tooling, you'd end up with a chicken and egg problem where someone needs to verify they got our tool to verify the binary.

Reply:
Update: 
* `gpg` supports `secp256k1`. https://wiki.gnupg.org/ECC
* Yubikey 5.2.3+ supports `secp256k1` as well! https://support.yubico.com/support/solutions/articles/15000027139-yubikey-5-2-3-enhancements-to-openpgp-3-4-support

With that, I will recommend in the doc I am writing now that developer public keys be made with that curve, which will allow them to be used as on-chain addresses in the future.